core/src/main/scala/org/apache/spark/shuffle/ShuffleManager.scala

```diff
@@ -36,8 +43,15 @@ private[spark] trait ShuffleManager {
       numMaps: Int,
       dependency: ShuffleDependency[K, V, C]): ShuffleHandle
 
-  /** Get a writer for a given partition. Called on executors by map tasks. */
-  def getWriter[K, V](handle: ShuffleHandle, mapId: Int, context: TaskContext): ShuffleWriter[K, V]
+  /** Get a writer for a given partition. Called on executors by map tasks.
```

Comment:
super nit: comment should follow javadoc formatting:

    /**
     *  Comment.
     */